pkg/api/norman/customization/cluster/validator.go

```diff
@@ -440,26 +443,52 @@ func validateEKSAccess(request *types.APIContext, eksConfig map[string]interface
 	return nil
 }
 
-func validateEKSNodegroups(request *types.APIContext, eksConfig map[string]interface{}, prevSpec *eksv1.EKSClusterConfigSpec) error {
-	noNodegroupsError := httperror.NewAPIError(httperror.InvalidBodyContent, fmt.Sprintf("must have at least one nodegroup"))
-	nodegroups, ok := eksConfig["nodeGroups"].([]interface{})
-	if !ok || nodegroups == nil {
-		if request.Method == http.MethodPost {
-			return noNodegroupsError
-		}
-		if prevSpec.NodeGroups != nil && len(prevSpec.NodeGroups) == 0 {
-			return noNodegroupsError
-		}
-	} else if len(nodegroups) == 0 {
-		return noNodegroupsError
+// validateEKSKubernetesVersion checks whether a kubernetes version is provided and if it is supported
+func validateEKSKubernetesVersion(eksConfig map[string]interface{}) error {
+	clusterVersion, _ := eksConfig["kubernetesVersion"]
+	if clusterVersion == nil {
+		return nil
```

Comment:
a nil defaults to some default version?

Reply:
nil means the field has not been managed through rancher yet and its effective value is its upstream value. The upstream value can be found on the status.